packages/salesforcedx-vscode-core/test/vscode-integration/commands/orgCreate.test.ts

```diff
@@ -125,17 +125,17 @@ describe('Force Org Create', () => {
       const CONFIG_FILE = 'configFile.txt';
       const TEST_ALIAS = 'testAlias';
       const TEST_ORG_EXPIRATION_DAYS = '7';
-      const forceOrgCreateBuilder = new ForceOrgCreateExecutor();
+      const forceOrgCreateBuilder = new OrgCreateExecutor();
       const createCommand = forceOrgCreateBuilder.build({
         file: path.join(workspaceUtils.getRootWorkspacePath(), CONFIG_FILE),
         alias: TEST_ALIAS,
         expirationDays: TEST_ORG_EXPIRATION_DAYS
       });
       expect(createCommand.toCommand()).to.equal(
-        `sfdx force:org:create -f ${CONFIG_FILE} --setalias ${TEST_ALIAS} --durationdays ${TEST_ORG_EXPIRATION_DAYS} --setdefaultusername --json --loglevel fatal`
+        `sfdx org:create:scratch --definition-file ${CONFIG_FILE} --alias ${TEST_ALIAS} --duration-days ${TEST_ORG_EXPIRATION_DAYS} --set-default --json`
```

Comment:
This command uses JSON, which is why I'm asking about `.withJson(false)` in the other command.